app/views/group_assignment_invitations/show.html.erb

```diff
@@ -19,21 +19,27 @@
 
   <hr>
 
-  <%= form_tag({ controller: :group_assignment_invitations, action: :accept_invitation}, method: :patch) do %>
-    <% if @groups.present? %>
-      <dt><label>Join an existing team</label></dt>
-      <dl class="form">
-        <dd><%= select :group, :id, options_for_select(@groups), include_blank: true %></dd>
-      </dl>
-    <% end %>
+  <% if @groups.present? %>
+    <p><label>Join an existing team</label></p>
+    <div class="team-select-grid">
+      <% @group_assignment.group_assignment_repos.each do |repo| %>
+          <%= render partial: 'group_assignment_team', locals: {repo: repo} %>
+      <% end %>
+      <div style="clear:both"></div>
+    </div>
+  <% end %>
+
+  <hr>
 
+  <%= form_tag({ controller: :group_assignment_invitations, action: :accept_invitation}, method: :patch) do %>
     <dl class="form">
       <dt><label><%= 'OR' if @groups.present? %> Create a new team</label></dt>
-      <dd><%= text_field_tag 'group[title]', nil, placeholder: 'Create a new team', class: 'textfield' %></dd>
+      <dd>
+        <%= text_field_tag 'group[title]', nil, placeholder: 'Create a new team', class: 'textfield' %>
+        <%= button_tag(type: 'submit', class: 'btn btn-green js-navigation js-form-submit') do %>
+          <i class="octicon octicon-plus"></i> Create Team
```

Comment:
So we have an octicon helper just for occasions like these.

```erb
<%= octicon('plus') %> Create team
```

If you could also lower case `Team` that would be ✨